### PR TITLE
Support setting of attitudeOrient via DeviceMotionData

### PR DESCRIPTION
**Pull request by IodaMikeMapbox**
Checks:

* [x] Update changelog
* [x] Rebase to `dev` branch
* [x] Assign reviewers

Linked issues:
mapbox/mapbox-vision#977

### CODE REVIEW

File: MapboxVision/Services/MotionManager.swift [hunk 1]
```diff
@@ -24,8 +24,9 @@ final class MotionManager {
         motion.showsDeviceMovementDisplay = true
         
         motion.startDeviceMotionUpdates(using: referenceFrame, to: queue) { [weak self] (data, error) in
-            guard let data = data else { return }
-            self?.handler?(data)
+            guard let data = data, let referenceFrame = self?.referenceFrame else { return }
```
Comment (chezzdev) [inline MapboxVision/Services/MotionManager.swift]: Consider taking `referenceFrame` from `motion.attitudeReferenceFrame` as an actual utilized value. We don't need to store it as a property then.

File: MapboxVision/Services/Constants.swift [hunk 2]
```diff
@@ -8,4 +9,6 @@ enum Constants {
     static let millisecondsInSecond: Double = 1000
     
     static let frameDuration: TimeInterval = 1 / 30.0
+
+    static let attitudeOrient: CMAttitudeReferenceFrame = .xArbitraryZVertical
```
Comment (dersim-davaod) [inline MapboxVision/Services/Constants.swift]: We don't need to have public constant here.
I suggest to have private enum in a local scope. Does that make sense?

File: MapboxVision/Services/MotionManager.swift (hunk 1 above)
Comment (IodaMikeMapbox) [reply MapboxVision/Services/MotionManager.swift]: done

File: MapboxVision/Services/Constants.swift (hunk 2 above)
Comment (dersim-davaod) [inline MapboxVision/Services/Constants.swift]: Why we choose `xArbitraryZVertical ` as a ref frame?
Did you consider roll and yaw for axes?

File: MapboxVision/Services/MotionManager.swift [hunk 3]
```diff
@@ -24,8 +24,9 @@ final class MotionManager {
         motion.showsDeviceMovementDisplay = true
         
         motion.startDeviceMotionUpdates(using: referenceFrame, to: queue) { [weak self] (data, error) in
-            guard let data = data else { return }
-            self?.handler?(data)
+            guard let data = data, let referenceFrame = self?.motion.attitudeReferenceFrame else { return }
```
Comment (dersim-davaod) [inline MapboxVision/Services/MotionManager.swift]: Do we need a local state?
Can we use `func startDeviceMotionUpdates(to: OperationQueue, withHandler: CMDeviceMotionHandler)`?

Comment (IodaMikeMapbox) [reply MapboxVision/Services/MotionManager.swift]: decided to not change

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,8 @@
 # Changelog
 
 ## 0.5.0 - Unreleased
-- Add support for UK country 
+  - Add support for UK country
```
Comment (chezzdev) [inline CHANGELOG.md]:
```suggestion
- Add support for UK country 
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,7 +1,8 @@
 # Changelog
 
 ## 0.5.0 - Unreleased
-- Add support for UK country 
+  - Add support for UK country 
+  - Support setting of attitudeOrient via DeviceMotionData
```
Comment (chezzdev) [inline CHANGELOG.md]:
```suggestion
- Support setting of attitudeOrient via DeviceMotionData
```

File: MapboxVision/Services/MotionManager.swift [hunk 6]
```diff
@@ -4,28 +4,25 @@ import CoreLocation
 
 final class MotionManager {
     private let motion: CMMotionManager = CMMotionManager()
-    private let referenceFrame: CMAttitudeReferenceFrame
+    private let referenceFrame: CMAttitudeReferenceFrame = .xArbitraryZVertical
```
Comment (chezzdev) [inline MapboxVision/Services/MotionManager.swift]: Please move it to high-level making it static const.

File: CHANGELOG.md (hunk 4 above)
Comment (IodaMikeMapbox) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 5 above)
Comment (IodaMikeMapbox) [reply CHANGELOG.md]: done

File: MapboxVision/Services/MotionManager.swift (hunk 6 above)
Comment (IodaMikeMapbox) [reply MapboxVision/Services/MotionManager.swift]: done

File: MapboxVision/Services/Constants.swift (hunk 2 above)
Comment (IodaMikeMapbox) [reply MapboxVision/Services/Constants.swift]: moved out here

Comment (IodaMikeMapbox) [reply MapboxVision/Services/Constants.swift]: I suppose 'cause it has the lowest CPU usage and we don't rely on that data

File: MapboxVision/Services/MotionManager.swift [hunk 7]
```diff
@@ -2,30 +2,28 @@ import Foundation
 import CoreMotion
 import CoreLocation
 
+private let referenceFrame: CMAttitudeReferenceFrame = .xArbitraryZVertical
```
Comment (dersim-davaod) [inline MapboxVision/Services/MotionManager.swift]: Could you please extract const into a static let within case-less enum?

**Comment by dersim-davaod** [comment]
> @IodaMikeMapbox Should we update usage of `sensors.setDeviceMotion` in DataProvider.swift?

UPD: no, we don't need to change it :)

File: MapboxVision/Services/MotionManager.swift (hunk 7 above)
Comment (IodaMikeMapbox) [reply MapboxVision/Services/MotionManager.swift]: done

**Comment by chezzdev** [comment]
Unblocked. Please rebase, solve conflicts and merge.

**Pull request merged**